### DISCUSSION TITLE
Add a thin bar underneath the send amount to visually convey how much of total supply is being sent.

### DISCUSSION THREAD

**Discussion by janimator0** [iotaledger/firefly#1554] [Features / Enhancements]
## Description

Would be comfortable to be able to have a thin bar underneath the send amount to help convey how much value you will be sent in comparison to the total amount. This will prevent mistakes (such as adding an extra zero to the value).

## Motivation

While it doesn't take long to do the math by looking up at your total supply and realize how much you would be spending there is the factor of user error. Someone (like my parents) would benefit greatly in knowing how much they would be spending, and if it makes sense.

## Requirements

As the user changes the value inside the Amounts input section within the "Send" screen the bar would grow and shrink relative to the total supply.

## Are you planning to do it yourself in a pull request?

No.

## Images of the feature
![image](https://user-images.githubusercontent.com/6241827/112145877-48b18d80-8bb1-11eb-8b94-6deaefa2a839.png)
![image](https://user-images.githubusercontent.com/6241827/112146196-aba32480-8bb1-11eb-95b3-a69738b32dca.png)

**Comment by rafaeldjpbrochado** [comment]
Great idea.

**Comment by cvarley100** [comment]
We can probably solve this problem by adding a send confirmation screen.

"You are about to send X Mi (= $Y)" "Cancel OK"
